examples/vhtlc.js

```diff
@@ -0,0 +1,224 @@
+// This example shows how to create a Virtual Hash Time Lock Contract (VHTLC)
+// and how to spend it.
+//
+// The VHTLC is a contract that allows Bob to claim a coin after 10 blocks if he reveals the preimage.
+// If Bob doesn't reveal the preimage, Alice can spend the VHTLC alone after 10 blocks.
```

Comment:
Preimage of what? I feel like it's put there out of context.. maybe better to use "secret"?
```suggestion
// The VHTLC is a contract that allows Bob to claim a coin after 10 blocks if he reveals a secret.
// If Bob doesn't reveal the secret, Alice can spend the VHTLC alone after 10 blocks.
```